packages/@lwc/integration-karma/scripts/karma-plugins/env.js

```diff
@@ -50,7 +50,8 @@ function createEnvFile() {
                 NATIVE_SHADOW_ROOT_DEFINED: typeof ShadowRoot !== 'undefined',
                 SYNTHETIC_SHADOW_ENABLED: ${SYNTHETIC_SHADOW_ENABLED},
                 LWC_VERSION: ${JSON.stringify(LWC_VERSION)},
-                API_VERSION: ${JSON.stringify(API_VERSION)}
+                API_VERSION: ${JSON.stringify(API_VERSION)},
+                ELEMENT_INTERNALS_DEFINED: typeof ElementInternals !== 'undefined'
```

Comment:
Using this to determine if the browser is compatible with `ElementInternals` and is only used on integration tests.

Reply:
@jmsjtu The way we have historically handled this kind of thing is to wrap the test in a runtime check:

```js
if (typeof ElementInternals !== 'undefined') {
  describe('element internals test', ...)
}
```

I think that's preferable to cluttering up the `env.js` file, plus it will naturally fix itself as we upgrade our test browsers.